libs/platform/src/lib/components/thumbnail/thumbnail-image/thumbnail-image.component.ts

```diff
@@ -26,6 +26,8 @@ export class ThumbnailImageComponent implements OnChanges, OnInit {
     @Output()
     thumbnailClicked: EventEmitter<Media> = new EventEmitter();
 
+    moreLabel: string;
```

Comment:
please add some doc/description for this field

Reply:
Modeified.